### PR TITLE
Sync authorities set

### PR DESCRIPTION
**Pull request by wuminzhe**

### CODE REVIEW

**Comment by HackFisher** [comment]
@wuminzhe CI failed.

**Comment by wuminzhe** [comment]
@HackFisher this fail is caused by `let accuracy = P::ACCURACY.saturated_into();`.
But the RUST_TOOLCHAIN is already `nightly-2020-10-06`, it is built correctly in my machine.

**Comment by wuminzhe** [comment]
I am trying to find why this happens.

File: .maintain/ropsten_crab_config.toml.sample [hunk 1]
```diff
@@ -40,6 +40,10 @@ topics = ['0xe77bf2fa8a25e63c1e5e29e1b2fcb6586d673931e020c4e3ffede453b830fb12']
 address = '0x49262B932E439271d05634c32978294C7Ea15d0C'
 topics = []
 
+[eth.contract.relay]
+address = '0xc8d6c331030886716f6e323ACB795077eB530E36'
```
Comment (HackFisher) [inline .maintain/ropsten_crab_config.toml.sample:44]: Add more comments pls

**Review by HackFisher — APPROVED** [review]

File: .maintain/ropsten_crab_config.toml.sample (hunk 1 above)
Comment (wuminzhe) [reply .maintain/ropsten_crab_config.toml.sample:44]: ok

**Pull request merged**